Add a workflow to bake blueprint properties of props (#6752)
FAForever/fa: changelog/snippets/other.6752.md

- (#6752) Add a GitHub Action workflow to bake properties into blueprints of props

> This enables us to reduce the blueprint processing during run time. That will reduce the loading time of users.

Reviewer: - Indentation is needed again after double newline to match the bullet point's indentation.

```suggestion
  This enables us to reduce the blueprint processing during run time, which will reduce the loading time for users.
```
